<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(://Fondo.jpg);
}
#Titulo{
color:white;}
#Tiempo{
color:white;}
#TxtPuntos{
color:white;}
#Score{
color:white;
qproperty-alignment: AlignCenter;
background: teal;
border: 3px solid silver;
border-radius: 7px;


}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>22</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Juego de Memoria</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tiempo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tiempo:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>651</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Tarjeta01">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta02">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta03">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta20">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta21">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta29">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta30">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta37">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta38">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta39">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta04">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta22">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta31">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta40">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta05">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta23">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta06">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta15">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta24">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta32">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta33">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta41">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta42">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta07">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta16">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta08">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta17">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta09">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta18">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>110</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta27">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta26">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta25">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta34">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta35">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta36">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>310</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta43">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Tarjeta44">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>410</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TxtPuntos">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>570</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
